src/v/model/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ v_cc_library(
     async_adl_serde.cc
     adl_serde.cc
     validation.cc
+    token_bucket.cc
```

Comment:
I was a little confused seeing a lot of changes in this commit, then i realized you also moved some source into a separate source file. I would break that out into another intermediate commit

Reply:
As it stands its not super easy to see what in this commit is specific to burst rate without manually diffing this commit with the previous one

Reply:
Yeah, sorry
Fixed it